src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -443,9 +443,7 @@ private static void NewMethod(out Construct obj1, out Construct obj2, out Constr
             await TestAsync(
 @"class Program { static void Main ( string [ ] args ) { [| (int, int) x = (1, 2); |]  System . Console . WriteLine ( x.Item1 ); } } " + TestResources.NetFX.ValueTuple.tuplelib_cs,
 @"class Program { static void Main ( string [ ] args ) { (int, int) x = {|Rename:NewMethod|}(); System.Console.WriteLine(x.Item1); } private static (int, int) NewMethod() { return (1, 2); } }" + TestResources.NetFX.ValueTuple.tuplelib_cs,
-index: 0,
-parseOptions: TestOptions.Regular,
-withScriptOption: true);
```

Comment:
Yes. Those options were needed previously because tuples required special parsing options, but that is no longer the case. So the defaults are just fine now.